compose.prod.yaml

```diff
@@ -91,6 +91,8 @@ x-airflow-common:
     AIRFLOW_VAR_SUL_PUB_HOST: ${AIRFLOW_VAR_SUL_PUB_HOST}
     AIRFLOW_VAR_SUL_PUB_KEY: ${AIRFLOW_VAR_SUL_PUB_KEY}
     AIRFLOW_VAR_DATA_DIR: /opt/airflow/data
+    AIRFLOW_VAR_GOOGLE_CONNECTION: google_cloud_default
+    AIRFLOW_VAR_GOOGLE_DRIVE_ID: ${AIRFLOW_VAR_GOOGLE_DRIVE_ID}
```

Comment:
this will come from vault (already there) and this Puppet PR: https://github.com/sul-dlss/puppet/pull/11998

folders setup:
stage: https://drive.google.com/drive/u/1/folders/1x2dTrafl986yIAMCBsOLoei2JHK4xatc
prod: https://drive.google.com/drive/u/1/folders/1G5xlN2OGuF-SRK_WN1OPgsroAx6jelan
dev (localhost dev): https://drive.google.com/drive/u/1/folders/1UchUau5bdErQbiD1btYpqP5SdVppIUFs